verify.rb

```diff
@@ -3,28 +3,28 @@
 require 'kwalify'
 @output = 0
 
-# YAML tags related to TFA
-@tfa_tags = {
-  # YAML tags for TFA Yes
-  true => %w[email hardware software sms phone doc],
-  # YAML tags for TFA No
-  false => %w[status twitter facebook email_address lang]
-}.freeze
-
 # Image max size (in bytes)
 @img_max_size = 2500
+#@img_max_size = 3000
```

Comment:
Any particular reason for the bump in max image size?

Reply:
Thats commented. I used that during my initial tests to see the rest of the verifications.

The hashtag means its commented